java/vector/src/main/java/org/apache/arrow/vector/compare/RangeEqualsVisitor.java

```diff
@@ -342,7 +342,7 @@ protected boolean compareFixedSizeListVectors(Range range) {
     }
 
     int listSize = leftVector.getListSize();
-    RangeEqualsVisitor innerVisitor = createInnerVisitor(leftVector, rightVector);
+    RangeEqualsVisitor innerVisitor = createInnerVisitor(leftVector.getDataVector(), rightVector.getDataVector());
```

Comment:
We do not have a test case for comparing fixed width list vector?
If so, can we add one?

Reply:
Thanks, you are right, we do not have FixedSizeList test and I just added one.

Reply:
Thanks for finding and fixing this bug.